Blockzilla/BrowserViewController.swift

```diff
@@ -467,35 +467,35 @@ class BrowserViewController: UIViewController {
         }
         
         // Screenshot the browser, showing the screenshot on top.
-        let screenshotView = view.snapshotView(afterScreenUpdates: true) ?? UIView()
-        
+        let image = mainContainerView.screenshot()
+        let screenshotView = UIImageView(image: image)
         mainContainerView.addSubview(screenshotView)
         screenshotView.snp.makeConstraints { make in
             make.edges.equalTo(mainContainerView)
         }
 
         clearBrowser()
         
-        UIView.animate(withDuration: UIConstants.layout.alphaToZeroDeleteAnimationDuration, animations: {
```

Comment:
Looks like you need to uncommit all these changes. I think there must have been a merge conflict here. This would all overwrite the changes I made in #1171